datayoga/blocks/add_field/block.py

```diff
@@ -11,11 +11,12 @@
 class Block(DyBlock):
     def init(self):
         logger.debug(f"Initializing {self.get_block_name()}")
-        self.expression = get_expression_class(self.properties["language"], self.properties["expression"])
+        self.expression = expression.compile(self.properties["language"], self.properties["expression"])
 
     def run(self, data: Any, context: Context = None) -> Any:
```

Comment:
data: Any should be replaced with data: List[Dict[str, Any]] and return type from Any to List[Dict[str, Any]].

Reply:
done